libsrc/telestrat/orixhdr.s

```diff
@@ -29,7 +29,7 @@
     .byte   $00                     ;  reserved
```

Comment:
Please remove the double blanks between the ";" and the comment text in this file.